app/decorators/proxy_rule_decorator.rb

```diff
@@ -4,11 +4,9 @@ class ProxyRuleDecorator < ApplicationDecorator
   self.include_root_in_json = false
 
   def pattern
-    pattern_value = object.pattern
-    return pattern_value unless backend_api_path
-    parts = ['/', backend_api_path]
-    parts << pattern_value unless pattern_value == '/'
-    File.join(*parts)
+    path_joined_parts = ['/', backend_api_path, object.pattern].join('/')
```

Comment:
```suggestion
    parts = [backend_api_path, object.pattern].compact
    '/' + parts.map { |part| StringUtils::StripSlash.strip_slash(part).presence }.compact.join('/')
```

Reply:
This suggestions is slightly different from what we had before. It removes the trailing slash in all cases, but the one where the mapping rule pattern is nothing but `/`.

Reply:
I liked it. wdyt @Martouta ?

Reply:
cool :+1: